core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -993,6 +999,12 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleApiVersionRequest(request: RequestChannel.Request) {
+    val responseHeader = new ResponseHeader(request.header.correlationId)
+    val responseBody = KafkaApis.apiVersionResponse
```

Comment:
Aren't we validating the version anywhere, to make sure we send an appropriate error?